### PR TITLE
feature/MatchFour

### PR DESCRIPTION
**Pull request by psk44**
This code iterates through the hand and finds if there is a match of 4 cards. Then it prints a statement of what you have.

### CODE REVIEW

**Review by kerenvr — APPROVED** [review]
Looks good to me. Very concise and easy to understand.

**Comment by psk44** [comment]
> Looks good to me. Very concise and easy to understand.

Thank you! Go fish! 🐟

**Review by RobertWheel — COMMENTED** [review]
Kinda hard to follow; maybe add comments on each of the for loops so that readers can follow, it could just be the complexity of the code and loops but dont think its a necessity, as long as you dont get lost in it. 👍
